atd-vzv/src/views/nav/SideMapControlDateRange.js

```diff
@@ -32,8 +34,8 @@ import {
 
 const SideMapControlDateRange = ({ type }) => {
   const [focused, setFocused] = useState(null);
-  const [start, setStart] = useState(mapStartDate);
-  const [end, setEnd] = useState(mapEndDate);
+  const [start, setStart] = useState(moment(mapStartDate));
+  const [end, setEnd] = useState(moment(mapEndDate));
```

Comment:
need to reconvert this into a moment object here for the `DateRangePicker` component since the rest of the changes i did turned this into a normal date object.

Reply:
Not sure exactly where this is coming from but I see that the UI is showing the "restore" icon when first visiting the map. iirc that icon should only show after the date start, end, or both are updated. When I click the icon in the preview, it breaks. Not my favorite situation that this picker relies on moment.js 😔 - thanks for updating around this limitation!

Default in staging/production
<img width="248" alt="Screenshot 2023-12-21 at 4 02 39 PM" src="https://github.com/cityofaustin/atd-vz-data/assets/37249039/4ca196c3-a66b-4bc6-92c5-783a4cabdbb7">

Default in preview
<img width="248" alt="Screenshot 2023-12-21 at 4 02 33 PM" src="https://github.com/cityofaustin/atd-vz-data/assets/37249039/2d759662-8a01-4e47-b3a8-2f32436c3343">

Reply:
Thanks for catching that! Just pushed up the fix